apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -129,6 +141,16 @@
                 <artifactId>guava</artifactId>
                 <version>${guava.version}</version>
             </dependency>
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
+                <version>${bytebuddy.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
```

Comment:
What are these for?

Reply:
find some conflict in agent-core.pom

Reply:
Could you show me the conflict?

Reply:
For this, you are just missing the dependency to dependecyManagement, which part have another dependency?

Reply:
OK, byte-buddy is rarely to trigger bugs usually. If you want to resolve, it is fine.

Reply:
just conflict on test scope

Reply:
![image](https://user-images.githubusercontent.com/31562192/97798395-98bea080-1c60-11eb-826c-860fb94fd42a.png)